app/src/services/cpk.ts

```diff
@@ -1360,11 +1360,13 @@ class CPKService {
         // get mainnet relay signer
         const to = await this.cpk.ethLibAdapter.signer.signer.getAddress()
 
+        const withdrawAmount = amount.sub(RELAY_FEE)
```

Comment:
Just gotta remove the relay fee here, must have accidentally stuck around in the merge.